docker/Dockerfile

```diff
@@ -190,13 +190,15 @@ RUN chown -R "${MAMBA_USER}:${MAMBA_USER}" "/home/${MAMBA_USER}/" && \
 
 COPY <<EOF /usr/local/bin/entrypoint.sh
 #!/bin/bash
+export USER=$(whoami)
```

Comment:
This is the fix. `$USER` and other vars are set by bash, and not guaranteed in Linux to exist. That's why Docker doesn't set those, and we didn't get `$USER` as that's done normally in an interactive env with the default bash settings + profile/bashrc/etc.